### ISSUE TITLE
Example response not complete for some YAMLs

### ISSUE THREAD

**Issue by janavMGX** [stoplightio/elements#2536]
## Context
We have revamped our API documentation and would like to use Stoplight as the new method of displaying the YAMLs.

## Current Behavior
The example body in the Try it panel doesn't seem to correspond to the attributes. [https://www.mediagenix.tv/api-j7d2XD7sFqcdWGeS/docs/Latest_version/Content/Content%20API/Stoplight%20API%20Explorer.htm#/paths/series/post](Seen in Series > POST series)
When I view the YAML in Swagger UI, the example body is a lot longer. [https://www.mediagenix.tv/api-j7d2XD7sFqcdWGeS/docs/Latest_version/Content/Content%20API/Swagger%20UI%20API%20Explorer.htm](Seen in Series > POST series)

It seems Stoplight is only showing the 

_The schema used in this call:_
    Series:
      allOf:
        - $ref: '#/components/schemas/Content'
        - type: object
          properties:
            contentId:
              type: string
            seasonIds:
              type: array
              items:
                type: string
                description: The contentId's of the seasons that are part of this series.
            contentType:
              type: string
              description: Maps to the drop-down PSIProductType
            productTitles:
              type: array
              items:
                $ref: '#/components/schemas/ProductTitle'
            contentCollectionIntentions:
              description: Maps to the drop-down PSIContentCollectionIntention
              type: array
              items:
                type: string           

_The result example body:_
{
  "contentId": "string",
  "seasonIds": [
    "string"
  ],
  "contentType": "string",
  "productTitles": [
    {
      "id": "string",
      "title": "string",
      "type": "string",
      "broadcastingTitle": false,
      "alternativeTitle": false
    }
  ],
  "contentCollectionIntentions": [
    "string"
  ]
}
It only seems to show the attributes directly defined on the schema and not the reference schema.

## Expected Behavior
Stoplight should also show the full example body including the referenced schema.

**Comment by weyert** [comment]
@janavMGX Could it be that your $ref'ed schema in your above schema is having `additionalProperties: false`? If you remove that does it work for you?

**Comment by janavMGX** [comment]
@weyert No, it only has properties defined like this:
    Content:
      properties:
        title:
          type: string
        duration:
        etc...

What I did try now is to move the $ref below the properties like this, and then it does show all of them. I'm not sure why that could be. Thanks!
`    Series:
      allOf:
        - type: object
          properties:
            contentId:
              type: string
            seasonIds:
              type: array
              items:
                type: string
                description: The contentId's of the seasons that are part of this series.
            contentType:
              type: string
              description: Maps to the drop-down PSIProductType
            productTitles:
              type: array
              items:
                $ref: '#/components/schemas/ProductTitle'
            contentCollectionIntentions:
              description: Maps to the drop-down PSIContentCollectionIntention
              type: array
              items:
                type: string
        - $ref: '#/components/schemas/Content'   `

**Comment by weyert** [comment]
Yeah, I have the same issue for when I removed the additionProperties: false in the referenced schema it started showing.

I haven't figured out what the cause is but if I deep down in @stoplight/json-schema-tree and enable ignoreAdditionalProperties then it shows up. 

So there is something going wrong when merging allOf entries.

**Comment by nawbc** [comment]
> Yeah, I have the same issue for when I removed the additionProperties: false in the referenced schema it started showing.
> 
> I haven't figured out what the cause is but if I deep down in `@stoplight/json-schema-tree` it uses a fork of `json-schema-merge-allof` (https://github.com/mokkabonna/json-schema-merge-allof) which has an setting `ignoreAdditionalProperties` then it shows up.
> 
> So there is something going wrong when merging allOf entries.

Any update?

**Comment by weyert** [comment]
@nawbc Not on my end. I have been busy working on work stuff. Might pick the sup at a later time. The code is not easy to understand for me so struggling a bit.

**Comment by weyert** [comment]
Same discussion: https://github.com/stoplightio/elements/issues/2593
